stories/rating-indicator/rating-indicator.stories.js

```diff
@@ -0,0 +1,529 @@
+            <label class="fd-rating-indicator__label" for="rating-half-sizes-35"></label>
+            
+            <input aria-label="4 star" type="radio" class="fd-rating-indicator__input" id="rating-half-sizes-4" name="rating-half-sizes" value="4">
+            <label class="fd-rating-indicator__label" for="rating-half-sizes-4"></label>
+            
+            <input aria-label="4 and half star" type="radio" class="fd-rating-indicator__input" id="rating-half-sizes-45" name="rating-half-sizes" value="4.5">
+            <label class="fd-rating-indicator__label" for="rating-half-sizes-45"></label>
+            
+            <input aria-label="5 star" type="radio" class="fd-rating-indicator__input" id="rating-half-sizes-5" name="rating-half-sizes" value="5">
+            <label class="fd-rating-indicator__label" for="rating-half-sizes-5"></label>
+        </div>
+        <span class="fd-rating-indicator__dynamic-text">(2.5 of 5)</span>
+    </div>
+</div>
+`;
+
+HalfSizes.parameters = {
+    docs: {
+        storyDescription: `
+To make the rating indicator to use half star needs to be added class \`.fd-rating-indicator--half-star\` 
+to the \`fd-rating-indicator\` element.
+`
+    }
+};
+
+export const CustomIcons = () => `
+<div class="example-container">
+    <div class="fd-rating-indicator is-icons"
+    style="--sapRating-indicator-icon-rated: url(${iconRated}); --sapRating-indicator-icon-unrated: url(${iconUnrated})">
+        <div class="fd-rating-indicator__container" aria-label="Star Rating (out of 5)">
+            <input aria-label="1 star" type="radio" class="fd-rating-indicator__input" id="rating-icon-1" name="rating-icon" value="1">
+            <label class="fd-rating-indicator__label" for="rating-icon-1"></label>
+            
+            <input aria-label="2 star" type="radio" class="fd-rating-indicator__input" id="rating-icon-2" name="rating-icon" value=2" checked>
+            <label class="fd-rating-indicator__label" for="rating-icon-2"></label>
+            
+            <input aria-label="3 star" type="radio" class="fd-rating-indicator__input" id="rating-icon-3" name="rating-icon" value="3">
+            <label class="fd-rating-indicator__label" for="rating-icon-3"></label>
+            
+            <input aria-label="4 star" type="radio" class="fd-rating-indicator__input" id="rating-icon-4" name="rating-icon" value="4">
+            <label class="fd-rating-indicator__label" for="rating-icon-4"></label>
+            
+            <input aria-label="5 star" type="radio" class="fd-rating-indicator__input" id="rating-icon-5" name="rating-icon" value="5">
+            <label class="fd-rating-indicator__label" for="rating-icon-5"></label>
+        </div>
+        <span class="fd-rating-indicator__dynamic-text">(2 of 5)</span>
+    </div>
+</div>
+`;
+
+CustomIcons.parameters = {
+    docs: {
+        storyDescription: `
+To make the rating indicator to use custom icons needs to be added class \`.fd-rating-indicator--icon\` 
```

Comment:
I'm not seeing how this is working in the code, perhaps it's more complex than this explanation?

Reply:
in the code updated this line
```html
<div class="fd-rating-indicator fd-rating-indicator--icon"
    style="--sapRating-indicator-icon-rated: url(${iconRated}); --sapRating-indicator-icon-unrated: url(${iconUnrated})">
```
if we go to the demo and look at the Custom icons section you will see smiles icons instead of star icon